app/models/server/models/Users.js

```diff
@@ -135,7 +135,11 @@ export class Users extends Base {
 		return this.findOne(query);
 	}
 
-	findOneOnlineAgentByUsername(username, options) { // TODO:: Create class Agent
+	findOneOnlineAgentByUserList(userList, options) { // TODO:: Create class Agent
```

Comment:
I found the method  **findOneOnlineAgentByUsername** [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/ee/app/livechat-enterprise/server/hooks/handleNextAgentPreferredEvents.js#L18) and [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/ee/app/livechat-enterprise/server/hooks/handleNextAgentPreferredEvents.js#L52). Shoud they be renamed too?